lib/rules/max-line-length/index.js

```diff
@@ -75,32 +116,21 @@ const rule = function(maxLength, options, context) {
       }
 
       const rawLineLength = nextNewlineIndex - match.endIndex;
+      const excludedLength = skippedSubStrings[skippedSubStringsIndex]
```

Comment:
compute excluded substring length (for current line), for instance:
```
url(
\t\t"bla bla"\u0020);
```
will give `excludedLength` equal to `"bla bla".length`

Reply:
it is a bit tricky since
```
url("bla \
bla \
");
```

will give `excludedLength` equals to `"bla \\\n".length` on line 1